data/models/nvd.py

```diff
@@ -26,35 +26,103 @@
 
 
 class Affect(nvd_template.Affect, NvdBase):
-    pass
+    def toJson(self):
```

Comment:
I know we're still inconsistent, but we would like to be pep8 compliant at some point 😅. So snakecase for naming would be great 🙂. (Same for the other methods below)